posthog-core/src/index.ts

```diff
@@ -478,6 +491,33 @@ export abstract class PostHogCore {
     return response
   }
 
+  getFeatureFlagPayload(key: string): JsonType | undefined {
+    const payloads = this.getFeatureFlagPayloads()
+
+    if (!payloads) {
+      return undefined
+    }
+
+    let response = payloads[key]
+
+    if (this.sendFeatureFlagEvent && !this.flagPayloadCallReported[key]) {
+      this.flagPayloadCallReported[key] = true
+      this.capture('$feature_flag_payload_called', {
```

Comment:
thoughts on consuming this into `$feature_flag_called` ? To a user, calling `getFeatureFlag`, or `getFeatureFlagPayload` seems equivalent. We can probs just add the payload parameter to feature_flag_called?

Also ensures no duplicate events sent when we call both `getFeatureFlag` and `getFeatureFlagPayload`

Reply:
Yes, but a little confused what the implementation should look like here. Would `getFeatureFlag ` and `getFeatureFlagPayload ` have the same exact logic now and just return different val since they both need to send the same event

Reply:
Hmm tbh, we can maybe get rid of this completely? Don't need to send these extra events 😅 . I kinda like the idea of everything being determined just by the variant, so there shouldn't be a need to audit the payload?

If you do decide to send them, yeah I was thinking the implementation looks the same, with the difference being `getFeatureFlag` returns the `$feature_flag_response` value while `getFeatureFlagPayload` returns the `$feature_flag_payload` 🤷 .

Reply:
Removed